backend/src/core/files.py

```diff
@@ -43,3 +44,32 @@ async def get_file(filename: str):
             filename=filename,
         )
     return None
+
+
+def image_cropping(file: UploadFile) -> str | None:
```

Comment:
This function is currently doing two things:
1. Cropping the `file`
2. Generating unique name & Saving the `file`

Could you please split this function into two smaller?

Also, here's a better name for a function doing point `1`:
```suggestion
def crop_image_to_square(file: UploadFile) -> str | None:
```